### ISSUE TITLE
l1 regularisation for C constraints

### ISSUE THREAD

**Issue by vicproon** [vicproon/whitomo#11]
Right now the dissimilarity constraint ($c_1 * c_2 = 0$) is implemented as soft additive quadratic regularisation to the loss: $loss = houghloss + ||c_1 * c_2||_2^2$
However L2 norm may be is too smooth, resulting in intermediate regularisation results. The use of L1 norm may produce better, more sharp responce.
